library/src/skins/default/src/sass/modules/tool/portal/_portal.scss

```diff
@@ -434,6 +439,18 @@ a[href]:not(.btn):hover {
   }
 }
 
+.site-opt-pin {
+  color: var(--sakai-pin-hover-color);
+}
+
+.bi-pin {
+  color: transparent;
+}
+
+.bi-pin-fill {
```

Comment:
We can use the same reasoning for si-pin though - if we modify it here it will affect all of Sakai. Doesn't this just need to be scoped to the nav and more sites sidebars?

Reply:
No, What I meant is if we were to use `bi-pin` somewhere else in Sakai, let's say we need it in red in someplace. how will you do it?
`pin-site: bi-pin (transparent color), other-pin: bi-pin(red)`.  And we add the class to `alias` but not the native.